README.md

```diff
@@ -1,17 +1,17 @@
-[![Build Status](https://travis-ci.org/src-d/gitbase-playground.svg)](https://travis-ci.org/src-d/gitbase-playground)
-[![codecov.io](https://codecov.io/github/src-d/gitbase-playground/coverage.svg)](https://codecov.io/github/src-d/gitbase-playground)
+[![Build Status](https://travis-ci.org/src-d/gitbase-web.svg)](https://travis-ci.org/src-d/gitbase-web)
+[![codecov.io](https://codecov.io/github/src-d/gitbase-web/coverage.svg)](https://codecov.io/github/src-d/gitbase-web)
```

Comment:
These seems to depend on repo name, but are ofc fine to break

Reply:
Yes, those will show as gray boxes, but it should still render.